test-docker-compose.yml

```diff
@@ -7,6 +7,9 @@ services:
 
   tinkerbell:
     build: .
+    command:
+      - --tls-cert=/certs/${FACILITY:-onprem}/bundle.pem
+      - --tls-key=/certs/${FACILITY:-onprem}/server-key.pem
```

Comment:
This was previously needed for the purposes of the old vagrant test, I'm not sure if this compose file is actually used anywhere now.